outputs.tf

```diff
@@ -0,0 +1,17 @@
+output "bucket" {
+  value = aws_s3_bucket.this
+
+  description = "The created bucket"
+}
+
+output "bucket_fullaccess_policy_document" {
```

Comment:
I'm not sure whether we should export a full access policy. Everyone using this module might have different requirements for the policies how to access the bucket.